src/layouts/SiteLaunchPad/components/SiteLaunchCheckList.tsx

```diff
@@ -306,16 +306,12 @@ export const SiteLaunchChecklistBody = ({
           <Table variant="simple">
             <Thead>
               <Tr>
-                <Td bg="gray.100" width="70%">
+                <Td bg="gray.100" width="100%">
                   <Text textStyle="subhead-2">
                     Site launch tasks ({tasksDone}/{numberOfTasks})
                   </Text>
                 </Td>
-                <Td bg="gray.100" textAlign="center" width="30%">
-                  <a href="mailto:support@isomer.gov.sg">
-                    I need support <Icon as={BxLifeBuoy} />
-                  </a>
-                </Td>
+                <Td bg="gray.100" width="30%" />
```

Comment:
Ideally this should not be needed, but when I tried 
A: 
![Screenshot 2023-08-18 at 1 55 14 PM](https://github.com/isomerpages/isomercms-frontend/assets/42832651/fdcdeb19-08a6-49aa-97eb-271463a3993b)

and 
B:
![Screenshot 2023-08-18 at 1 56 13 PM](https://github.com/isomerpages/isomercms-frontend/assets/42832651/de360641-81c3-4f13-b130-d16fffa63a9d)

both did not give me the full gray width that I wanted

![Screenshot 2023-08-18 at 1 54 24 PM](https://github.com/isomerpages/isomercms-frontend/assets/42832651/5cbf7f8a-fca4-494a-b316-aba20757861b)

Reply:
i think is because the table body is divided into 2 hence chakra interprets the same for the heading